src/app/components/sidebar/categories/Categories.test.tsx

```diff
@@ -0,0 +1,59 @@
+import { theme } from 'assets/styles/theme.styles';
+import { render } from 'tests';
+import { CATEGORY_MOCK } from 'tests/categoriesMock';
+import { Categories } from './Categories';
+
+describe('Categories component', () => {
+  it('renders "Categories" header', () => {
```

Comment:
Generalnie, bardzo mi się podoba w jaki sposób testujesz komponenty - moim zdaniem te test casy są trafione.
Niemniej brakuje mi opisu, w jakim przypadku jest renderowane to co jest renderowane. Lecąc po kolei nazywałbym te testy tak:
```
renders Categories header correctly
renders all categories with count when none selected
<ten jest ok>
renders selected category correctly
<to sprawdzanie wrzuciłbym do poprzedniego scenariusza, bo nazwa pokrywa oba casy>
fires "all machines" callback correctly
```